### DISCUSSION TITLE
doris

### DISCUSSION THREAD

**Discussion by foole11** [apache/doris#26747] [A - General / Q&A]
Execution failed: Error Failed to execute sql: java.sql.SQLException: (conn=27354) errCode = 2, detailMessage = Failed to create repository: failed to close writer, msg:File /test/2023-11-01_1698983396/__palo_repository_mydi_repo_1698983396_11/__repo_info could only be written to 0 of the 1 minReplication nodes. There are 0 datanode(s) running and 0 node(s) are excluded in this operation. at org.apache.hadoop.hdfs.server.blockmanagement.BlockManager.chooseTarget4NewBlock(BlockManager.java:2350) at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.chooseTargetForNewBlock(FSDirWriteFileOp.java:294) at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2989) at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:912) at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:595) at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java) at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:621) at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:589) at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:573) at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1227) at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1094) at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1017) at java.security.AccessController.doPrivileged(Native Method) at javax.security.auth.Subject.doAs(Subject.java:422) at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1899) at org.apache.hadoop.ipc.Server$Handler.run(Server.java:3048)

    CREATE REPOSITORY `mydi_repo_1698983396_11`
    WITH hdfs
    ON LOCATION "hdfs://192.168.101.xx:9000/test/2023-11-01_1698983396"
    PROPERTIES
    (
        "fs.defaultFS"="hdfs://192.168.101.xx:9000",
        "hadoop.username"="root",
        "hadoop.pass_word"="root"
    );

hdfs是远程单节点部署。

**Comment by LemonLiTree** [comment]
0 datanode(s) running？？
